packages/office-ui-fabric-react/src/components/Callout/CalloutContent.styles.ts

```diff
@@ -75,8 +68,7 @@ export const getStyles = (props: ICalloutContentStyleProps): ICalloutContentStyl
       },
       focusClear(),
       className,
-      positions && positions.targetEdge && ANIMATIONS[positions.targetEdge],
-      !!calloutWidth && { width: calloutWidth },
+      !!calloutWidth && { width: calloutWidth }
```

Comment:
Remove positions const (line 38). I believe it is no longer being used. I'd remove it from the props, but it is probably a breaking change.